src/woo_publications/metadata/api/filters.py

```diff
@@ -0,0 +1,29 @@
+from django.utils.translation import gettext_lazy as _
+
+from django_filters.rest_framework import FilterSet, filters
+
+from woo_publications.api.filters import URLFilter
+
+from ..models import InformationCategory
+
+
+class InformationCategoryFilterset(FilterSet):
+    identifier = URLFilter(
+        lookup_expr="icontains",
+        help_text=_(
+            "Search the information category based on the unique IRI that identifies a specific category."
+        ),
+    )
```

Comment:
I only remarked the partial match for the `naam`, the identifier should be an exact match. Nobody will every enter an identifier like this in a search box in the UI.

This should be `lookup_expr="exact"`